frontend/src/scenes/actions/ActionEdit.js

```diff
@@ -11,6 +11,7 @@ export function ActionEdit({ actionId, apiURL, onSave, user, isEditor, simmer, s
         id: actionId,
         apiURL,
         onSave: (action, createNew) => onSave(action, !actionId, createNew),
+        temporaryToken,
         params: '?include_count=1' + (temporaryToken ? '&temporary_token=' + temporaryToken : ''),
```

Comment:
We should pass through only one instance of temporary token. I originally checked on this file and overlooked it because I thought the params were reused in the logic

Reply:
Good point, fixed